### PR TITLE
Precommit setup, functional problem fixes

### PR DESCRIPTION
**Pull request by mandrono**

### CODE REVIEW

File: inference-engine/src/mkldnn_plugin/nodes/mkldnn_normalize_node.cpp [hunk 1]
```diff
@@ -686,12 +688,13 @@ bool MKLDNNNormalizeL2Node::isSupportedOperation(const std::shared_ptr<ngraph::N
             }
             return false;
         };
-        if (!isSupportedAxes(axesNode->cast_vector<size_t>(), dataDims)) {
-            errorMessage = errorPrefix + "supports only per channel or per channel and per spatial reduction";
+        if (!isSupportedAxes(axesNode->cast_vector<size_t>(), dataDims) && ngraph::shape_size(axesNode->get_shape()) != 0) {
+            errorMessage = "Supports only per channel, per channel and per spatial, no reduction cases";
```
Comment (dmitry-gorokhov) [inline inference-engine/src/mkldnn_plugin/nodes/mkldnn_normalize_node.cpp:692]: per channel, per channel?

File: inference-engine/src/mkldnn_plugin/nodes/mkldnn_normalize_node.cpp [hunk 2]
```diff
@@ -648,8 +648,11 @@ MKLDNNNormalizeL2Node::MKLDNNNormalizeL2Node(const std::shared_ptr<ngraph::Node>
     std::string errorMessage;
     if (isSupportedOperation(op, errorMessage)) {
         errorPrefix = "NormalizeL2 node with name '" + getName() + "' ";
-        eps = std::dynamic_pointer_cast<const ngraph::op::v0::NormalizeL2>(op)->get_eps();
+        const auto norm = std::dynamic_pointer_cast<const ngraph::op::v0::NormalizeL2>(op);
+        eps = norm->get_eps();
+        epsMode = norm->get_eps_mode() == ngraph::op::EpsMode::MAX ? NormEpsMode::MAX : NormEpsMode::ADD;
         across_spatial = ngraph::shape_size(op->get_input_shape(AXES)) != 1;
+        cornerCase = op->get_input_shape(AXES).empty();
```
Comment (dmitry-gorokhov) [inline inference-engine/src/mkldnn_plugin/nodes/mkldnn_normalize_node.cpp:655]: Could you please add comment what actually cornerCase means. (Can be copied from the spec)

File: inference-engine/src/mkldnn_plugin/nodes/mkldnn_normalize_node.h [hunk 3]
```diff
@@ -89,6 +89,24 @@ class MKLDNNNormalizeL2Node : public MKLDNNNode {
     bool canFuse(const MKLDNNNodePtr& node) const override;
 
 private:
+    enum class NormEpsMode {
+        ADD,
+        MAX
+    };
+    NormEpsMode epsMode = NormEpsMode::ADD;
+
+    float epsApply(const float &modulo) {
```
[the remark this answers is not in the document]
Comment (mandrono) [reply inference-engine/src/mkldnn_plugin/nodes/mkldnn_normalize_node.h]: fixed

File: inference-engine/src/mkldnn_plugin/nodes/mkldnn_normalize_node.cpp (hunk 1 above)
Comment (mandrono) [reply inference-engine/src/mkldnn_plugin/nodes/mkldnn_normalize_node.cpp]: print list unsupported axes

File: inference-engine/src/mkldnn_plugin/nodes/mkldnn_normalize_node.cpp (hunk 2 above)
Comment (mandrono) [reply inference-engine/src/mkldnn_plugin/nodes/mkldnn_normalize_node.cpp:657]: added from spec

**Review by dmitry-gorokhov — APPROVED** [review]

**Pull request merged**